packages/eslint-config-react-native-community/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@react-native-community/eslint-config",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
shouldn't this be a more "important" bump? Like, adding support for ESLint 8 seems something that should be at least a minor bump if not even a major 🤔

Reply:
This is not a breaking change but maybe 3.1.0?